lib/permissions.android.js

```diff
@@ -8,16 +8,31 @@ type CheckOptions = string | { type: string }
 type RequestOptions = string | { type: string, rationale?: Rationale }
 
 const permissionTypes = {
+  coarseLocation: PermissionsAndroid.PERMISSIONS.ACCESS_COARSE_LOCATION,
   location: PermissionsAndroid.PERMISSIONS.ACCESS_FINE_LOCATION,
   camera: PermissionsAndroid.PERMISSIONS.CAMERA,
   microphone: PermissionsAndroid.PERMISSIONS.RECORD_AUDIO,
   contacts: PermissionsAndroid.PERMISSIONS.READ_CONTACTS,
+  writeContacts: PermissionsAndroid.PERMISSIONS.WRITE_CONTACTS,
   event: PermissionsAndroid.PERMISSIONS.READ_CALENDAR,
+  writeEvent: PermissionsAndroid.PERMISSIONS.WRITE_CALENDAR,
+  accounts: PermissionsAndroid.PERMISSIONS.GET_ACCOUNTS,
+  readStorage: PermissionsAndroid.PERMISSIONS.READ_EXTERNAL_STORAGE,
   storage: PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE,
   photo: PermissionsAndroid.PERMISSIONS.WRITE_EXTERNAL_STORAGE,
```

Comment:
@yonahforst should this be asking for READ_EXTERNAL_STORAGE instead of WRITE_EXTERNAL_STORAGE?